### ISSUE TITLE
Can't Copy and Paste from Variable explorer

### ISSUE THREAD

**Issue by zsinx6** [spyder-ide/spyder#8206]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

Can't copy and paste from Variable explorer

### What steps reproduce the problem?

1. Create a file with simple code, as follows
```
 foo = ["Test1", "Test2"]
```
2. Save and run the file
3. Open variable explorer then double click in the list's values
4. Select any value then right click on it
5. Choose Copy
6. Right click in the background of the new window, then choose Paste

### What is the expected output? What do you see instead?
The expected output should be the copied item to be pasted. But an errors occurs and nothing happens.

### Paste Traceback/Error Below (if applicable)

```python-traceback

  File "/usr/lib/python3.7/site-packages/spyder/widgets/variableexplorer/collectionseditor.py", line 1163, in paste
    self.import_from_string(cliptext, title=_("Import from clipboard"))
  File "/usr/lib/python3.7/site-packages/spyder/widgets/variableexplorer/collectionseditor.py", line 1150, in import_from_string
    blacklist=list(data.keys())))
AttributeError: 'list' object has no attribute 'keys'

```

## Versions

* Spyder version: 3.3.1
* Python version: Python 3.7.1 64bits
* Qt version: Qt 5.9.3
* PyQt version: PyQt5 5.9.2
* Operating System name/version: Arch Linux 4.18.16-arch1-1-ARCH

### Dependencies

```
IPython >=4.0     :  6.5.0 (OK)
cython >=0.21     :  None (NOK)
jedi >=0.9.0      :  0.13.1 (OK)
matplotlib >=2.0.0:  None (NOK)
nbconvert >=4.0   :  5.4.0 (OK)
numpy >=1.7       :  None (NOK)
pandas >=0.13.1   :  None (NOK)
psutil >=0.3      :  5.4.7 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pyflakes >=0.6.0  :  2.0.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
pylint >=0.25     :  2.1.1 (OK)
qtconsole >=4.2.0 :  4.4.1 (OK)
rope >=0.9.4      :  0.11.0 (OK)
sphinx >=0.6.6    :  1.8.0 (OK)
sympy >=0.7.3     :  None (NOK)
```

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll take a look at this problem in a future release.

**Comment by ccordoba12** [comment]
This was fixed in our 3.3.2 version.

**Comment by evemilano** [comment]
> This was fixed in our 3.3.2 version.

Fixed? I don't think so. in 64 bit I still have this embarrassing problem...
